community-plugins.json

```diff
@@ -4831,5 +4831,13 @@
         "description": "Adds a button to the search view for copying the Obsidian search URL.",
         "repo": "czottmann/obsidian-copy-search-url",
         "branch": "main"
+    },
+    {
+        "id": "obsidian-keyboard-analyzer"
+        "name": "Keyboard Analyzer",
+        "author": "Cogscieds",
+        "description": "See and analyse your keyboard hotkeys and shortcuts"
```

Comment:
There is a comma missing here, blocking our validation script from running.
This would also block Obsidian itself from showing any plugins if this where merged.